flatpak/theia.sh

```diff
@@ -0,0 +1,42 @@
+#!/bin/sh
+
+# It is important to set --electronUserData and THEIA_CONFIG_DIR
+# to the same paths in both the sandbox and the native run! Otherwise
+# settings and workspaces won't be shared.
+# Since this is a Flatpak, everything will be stored within the Flatpak directory structure.
+
+default_args="--ozone-platform-hint=auto --electronUserData=$XDG_DATA_HOME"
```

Comment:
The electronUserData flag does not seem to be necessary when passing the XDG envs as as a whole:
```
--env=XDG_DATA_HOME="${XDG_DATA_HOME:-}" \
--env=XDG_CONFIG_HOME="${XDG_CONFIG_HOME:-}" \
--env=XDG_CACHE_HOME="${XDG_CACHE_HOME:-}" \
 --env=XDG_STATE_HOME="${XDG_STATE_HOME:-}" \
```

Not sure what the preferred way is here.